docker-compose-example.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.3"
+volumes:
+  logs:
```

Comment:
```suggestion
```

Reply:
Why?

Honestly curious.

It makes it a little easier to know which volume is being used if you use docker compose.

For example (my system);

```
docker volume ls
DRIVER    VOLUME NAME
local     aa1286298e34c694616d50cbc6cf1524fad0dff250a4ff36bf0f105b72d592ca
local     d523a981f0f6acd1b84ffc3c60188a2e09daf4a64f0910f5eaf4c998f664700d
local     jellyfin_data
local     stationeers_logs
```

At a glance I have no idea which volume `aa1286298e34c694616d50cbc6cf1524fad0dff250a4ff36bf0f105b72d592ca` is assigned to while I'm in cli.  But I know **exactly** what volume `stationeers_logs` belongs to.

Docker compose creates the volume name (when defined) with <services>_<volume>.  In that docker compose the service that is using `logs` is `stationeers` - hence the volume being `stationeers_logs` on my system.

I'll still remove it if you still think it shouldn't be there :smiley: